### PR TITLE
145 update from easyCore to easyscience

### PR DESCRIPTION
**Pull request by andped10**
Code now use `easyscience` rather than the deprecated `easyCode` package

### CODE REVIEW

File: EasyReflectometry/calculators/bornagain/wrapper.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 __author__ = 'github.com/arm61'
 
 import bornagain as ba
-from easyCore import np
+from easyscience import np
```
Comment (rozyczko) [inline EasyReflectometry/calculators/bornagain/wrapper.py:4]: I think we removed the `easycore` version of numpy and moved to
```python
import numpy as np
```

File: EasyReflectometry/calculators/refl1d/wrapper.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 from typing import Tuple
 
-from easyCore import np
+from easyscience import np
```
Comment (rozyczko) [inline EasyReflectometry/calculators/refl1d/wrapper.py:5]: same as above

**Review by rozyczko — APPROVED** [review]
Looks good except for a few occurences of the deprecated `numpy` import

File: EasyReflectometry/calculators/refl1d/wrapper.py (hunk 2 above)
Comment (andped10) [reply EasyReflectometry/calculators/refl1d/wrapper.py:5]: Thanks for pointing this out.  I forgot.

Comment (andped10) [reply EasyReflectometry/calculators/refl1d/wrapper.py]: Did a search and replace in repo

File: EasyReflectometry/calculators/bornagain/wrapper.py (hunk 1 above)
Comment (andped10) [reply EasyReflectometry/calculators/bornagain/wrapper.py]: Thanks

**Pull request merged**